config.toml.sample

```diff
@@ -9,6 +9,12 @@
     admin_username = "listmonk"
     admin_password = "listmonk"
 
+[tls]
+    enable = true
```

Comment:
Please default this to "off" in the sample config. This feature should be explicitly requested by the admin, not assumed. It is too intrusive and can mess up other domain certificate arrangements.